### PR TITLE
Created /admin/queues endpoint

### PR DESCRIPTION
**Pull request by cipatera**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
created /admin/queues endpoint, added queue elements to queue model, added queue elements to /queue/create
- **What is the current behavior?** (You can also link to an open issue here)

- **What is the new behavior (if this is a feature change)?**

- **Other information**:

### CODE REVIEW

File: packages/backend/src/controllers/admin.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { IQueue, IAdmin } from "@waiting-line-app/shared-dto/db";
+
+const getAdminQueue = async (adminId: string) => {
+  const admin: IAdmin = await Queue.find({ adminId });
+  if (!admin) {
+    throw new Error("admin not found");
+  }
+  const arrOfQueueProperties: IQueue = [];
+  for (let i = 0; i < admin.length; i++) {
+    const queueName: string = admin[i].queueName;
```
Comment (jchen42703) [inline packages/backend/src/controllers/admin.ts:14]: Typescript implicitly types! Please do not explicitly write the types if you do not have to. (It's not like java)

File: packages/backend/src/controllers/admin.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { NextFunction, Request, Response, Router } from "express";
+import { Queue } from "../lib/models/queue";
+import { HttpException } from "../lib/errors";
+import { GETQueueReq, GETQueueRes } from "@waiting-line-app/shared-dto/admin";
+import { IQueue, IAdmin } from "@waiting-line-app/shared-dto/db";
+
+const getAdminQueue = async (adminId: string) => {
```
Comment (jchen42703) [inline packages/backend/src/controllers/admin.ts:7]: move this function to queue.service.ts and change the name to getAllQueuesForAdmin or something clearer

File: packages/shared-dto/admin.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { IQueue } from "@waiting-line-app/shared-dto/db";
```
Comment (jchen42703) [inline packages/shared-dto/admin.ts:1]: please move this to /shared-dto/src

Comment (jchen42703) [reply packages/shared-dto/admin.ts:1]: also make sure to add the admin interfaces to the exports of index.ts

**Review by jchen42703 — CHANGES_REQUESTED** [review]
Small changes but overall looks good. I didn't run, but I'll assume it works!

File: packages/backend/src/controllers/admin.ts (hunk 1 above)
Comment (cipatera) [reply packages/backend/src/controllers/admin.ts:14]: Which line are you referring to? When I hovered my mouse over `queueName = admin[i].queueName`, queueName was type any. But I'll change it back

**Comment by cipatera** [comment]
I performed the changes

**Comment by jchen42703** [comment]
seal_of_approval

**Pull request merged**